testgrid/config/config.yaml

```diff
@@ -3936,6 +3936,517 @@ dashboards:
+    column_header:
+    - configuration_value: node_os_image
+    - configuration_value: Commit
+    - configuration_value: infra-commit
+  - name: gci-gke-multizone
+    test_group_name: ci-kubernetes-e2e-gci-gke-multizone
+    column_header:
+    - configuration_value: node_os_image
+    - configuration_value: Commit
+    - configuration_value: infra-commit
+  - name: gci-gke-reboot
+    test_group_name: ci-kubernetes-e2e-gci-gke-reboot
+    column_header:
+    - configuration_value: node_os_image
+    - configuration_value: Commit
+    - configuration_value: infra-commit
+  - name: gci-gke-subnet
+    test_group_name: ci-kubernetes-e2e-gci-gke-subnet
+    column_header:
+    - configuration_value: node_os_image
+    - configuration_value: Commit
+    - configuration_value: infra-commit
+  - name: gci-gke-test
+    test_group_name: ci-kubernetes-e2e-gci-gke-test
+    column_header:
+    - configuration_value: node_os_image
+    - configuration_value: Commit
+    - configuration_value: infra-commit
+  - name: gci-gke-updown
+    test_group_name: ci-kubernetes-e2e-gci-gke-updown
+    column_header:
+    - configuration_value: node_os_image
+    - configuration_value: Commit
+    - configuration_value: infra-commit
+  - name: gke-device-plugin-gpu
+    test_group_name: ci-kubernetes-e2e-gke-device-plugin-gpu
+    column_header:
+    - configuration_value: node_os_image
+    - configuration_value: Commit
+    - configuration_value: infra-commit
+  - name: gke-device-plugin-gpu-p100
+    test_group_name: ci-kubernetes-e2e-gke-device-plugin-gpu-p100
+    column_header:
+    - configuration_value: node_os_image
+    - configuration_value: Commit
+    - configuration_value: infra-commit
+  - name: gke-canary
+    test_group_name: ci-kubernetes-e2e-gke-canary
+    column_header:
+    - configuration_value: node_os_image
+    - configuration_value: Commit
+    - configuration_value: infra-commit
+  - name: e2e-gke-cosbeta-k8sdev-soak
+    test_group_name: ci-kubernetes-e2e-gke-cosbeta-k8sdev-soak
+    column_header:
+    - configuration_value: node_os_image
+    - configuration_value: Commit
+    - configuration_value: infra-commit
+  - name: e2e-gke-cosbeta-k8sdev-stackdriver
+    test_group_name: ci-kubernetes-e2e-gke-cosbeta-k8sdev-stackdriver
+    column_header:
+    - configuration_value: node_os_image
+    - configuration_value: Commit
+    - configuration_value: infra-commit
+  - name: e2e-gke-cosbeta-k8sbeta-soak
+    test_group_name: ci-kubernetes-e2e-gke-cosbeta-k8sbeta-soak
+    column_header:
+    - configuration_value: node_os_image
+    - configuration_value: Commit
+    - configuration_value: infra-commit
+  - name: e2e-gke-cosbeta-k8sbeta-stackdriver
+    test_group_name: ci-kubernetes-e2e-gke-cosbeta-k8sbeta-stackdriver
+    column_header:
+    - configuration_value: node_os_image
+    - configuration_value: Commit
+    - configuration_value: infra-commit
+
+- name: sig-gcp-release-1.8
+  dashboard_tab:
+  - name: gci-gke-1.8
```

Comment:
bikeshed: I'll unify the rest of release-foo and stable-foo jobs, not sure here you want to keep release version and update each release, or just use `-stable-foo` instead?

Reply:
I'd like to name the groups as `dev`, `beta` and `stable` but the included jobs do not all use this naming.